templates/index.html

```diff
@@ -11,8 +11,26 @@ <h1 class="display-4">To-Do App</h1>
     <div class="col-md-12">
       <h2>Items</h2>
       <ul class="list-group mb-4">
-        <!-- List the todo items here -->
+        {% for task in tasks %}
+          <li>
+            {% if task.status == "Completed" %}
+            &check;
```

Comment:
Minor observation about indentation (this should be in a level). Same for the else section below